Objective-C/Internal/Replicator/CBLWebSocket.mm

```diff
@@ -467,10 +470,11 @@ - (void)URLSession:(NSURLSession *)session writeClosedForStreamTask:(NSURLSessio
 }
 
 - (void) streamClosed: (BOOL)isWrite {
+    BOOL expectedClose = _requestedClose || _closeOnError;
     CBLLog(WebSocket, @"CBLWebSocket %s stream closed%s",
            (isWrite ? "write" : "read"),
-           (_requestedClose ? "" : " unexpectedly"));
-    if (!_requestedClose) {
+           (expectedClose ? "" : " unexpectedly"));
```

Comment:
By saying "unexpectedly" at the end, it feels easy to overlook.  What about somehow putting "WARNING" in the message in this case to make it more apparent?

Maybe:

```
 (expectedClose ? "" : " WARNING: connection closed unexpectedly"));
```

although .. if it happens often, then it's extra noise.